src/location/tide-phrase/index.js

```diff
@@ -24,12 +23,19 @@ const TidePhrase = class extends Component {
   }
 }
 
-const tideDirection = nextTide => {
-  if (nextTide.type === 'high') {
-    return _.upperFirst('incoming')
+const cityName = userLocation => {
+  if (userLocation) {
+    return geoCodeLocation(userLocation)
+      .then(city => {
+        this.setState({ city: city })
+        return city
```

Comment:
I think this is my issue. I need to be setting state here, not returning anything since this is a promise. 

I also need to kick off this function when it loads (but I need it to keep triggering in case of a state change from elsewhere.

`didMount` would only run once right? 

Then on line 17 I would be using `{this.state.city}`

Reply:
yep

Reply:
<img width="467" alt="screen shot 2018-02-28 at 2 44 10 pm" src="https://user-images.githubusercontent.com/1182341/36809412-df34e4c0-1c95-11e8-83eb-ac8e01a4cf45.png">